virtual_ecosystem/models/abiotic_simple/constants.py

```diff
@@ -31,3 +31,37 @@ class AbioticSimpleConsts(ConstantsDataclass):
     """factor 2 for saturation vapour pressure calculation."""
     saturation_vapour_pressure_factor3: float = 237.3
     """factor 3 for saturation vapour pressure calculation."""
+
+
+@dataclass(frozen=True)
+class AbioticSimpleBounds(ConstantsDataclass):
+    """Upper and lower bounds for abiotic variables.
+
+    When a values falls outside these bounds, it is set to the bound value. Note that
+    this approach does not conserve energy and matter in the system. This will be
+    implemented at a later stage.
+    """
```

Comment:
I think this would be cleaner as tuples:

```python

@dataclass(frozen=True)
class AbioticSimpleBounds(ConstantsDataclass):
    air_temperature: tuple[float, float] = (-20.0, 80.0)
```

It's more concise and makes it easier to unpack the data when you need it.

Reply:
This is a good suggestion. I added the gradients to the tuples, too, so it's a nice package for each variable.

Reply:
I did wonder about including the gradients too - not everything has a gradient, so I kept it to bounds, but I think that works too.